### PR TITLE
Add page for react-native-config component

### PR DESCRIPTION
**Pull request by tritao**
ApplicationFrameHost_Eu84ikMZta

### CODE REVIEW

File: src/RNGalleryList.ts [hunk 1]
```diff
@@ -3,12 +3,14 @@ import React from 'react';
...
 import {TemplateExamplePage} from './examples/TemplateExamplePage';
 import {CheckBoxExamplePage} from './examples/CheckBoxExamplePage';
+import {ConfigExamplePage} from './examples/ConfigExamplePage';
 import {DatePickerExamplePage} from './examples/DatePickerExamplePage';
 import {TimePickerExamplePage} from './examples/TimePickerExamplePage';
 import {SliderExamplePage} from './examples/SliderExamplePage';
 import {PickerExamplePage} from './examples/PickerExamplePage';
 import {DeviceInfoExamplePage} from './examples/DeviceInfoExamplePage';
 
+
```
Comment (chiaramooney) [inline src/RNGalleryList.ts:13]: nit: Can you delete this extra new line?

**Comment by chiaramooney** [comment]
PR just merged which adds a hyperlink section at the bottom of each component page with links to documentation. See the template page as an example. Could you add that section to this page as well?

**Comment by tritao** [comment]
Updated.

File: src/examples/ConfigExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+  return (
+    <Page
+      title="Config Variables"
+      description="Shows exposed config variables via the react-native-config module.">
+
+      <View >
+        <Text>Keep in mind this module doesn't obfuscate or encrypt secrets for packaging, so do not store sensitive keys in .env.</Text>
+      </View>
+
```
Comment (chiaramooney) [inline src/examples/ConfigExamplePage.tsx:34]: Could you move this statement inside of the description prop?

**Review by chiaramooney — CHANGES_REQUESTED** [review]
One last layout change, otherwise looks great!

**Comment by tritao** [comment]
Updated.

**Review by chiaramooney — APPROVED** [review]
👍

**Pull request merged**